app/src/main/res/layout/activity_event_management.xml

```diff
@@ -6,4 +6,277 @@
+                android:paddingStart="10dp"
+                android:paddingEnd="0dp"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toBottomOf="@id/eventManagementDescriptionLayout">
+
+                <TextView
+                    android:id="@+id/tvSpinnerChoice"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:layout_weight="2.5"
+                    android:text="@string/choose_a_zone_hint"
+                    android:textAppearance="@style/TextAppearance.AppCompat.Body2" />
+
+                <Spinner
+                    android:id="@+id/spinner_zone"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:layout_weight="1" />
+            </LinearLayout>
+
+            <androidx.constraintlayout.widget.ConstraintLayout
+                android:id="@+id/constraint_layout_date_start"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="10dp"
+                android:padding="0dp"
+                android:paddingStart="10dp"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintTop_toBottomOf="@+id/limitedEventLayout"
+                tools:ignore="RtlSymmetry">
+
+                <LinearLayout
+                    android:id="@+id/start_date_text"
+                    android:layout_width="wrap_content"
+                    android:layout_height="0dp"
+                    android:orientation="vertical"
+                    app:layout_constraintBottom_toBottomOf="parent"
+                    app:layout_constraintStart_toStartOf="parent"
+                    app:layout_constraintTop_toTopOf="parent">
+
+                    <TextView
+                        android:id="@+id/tvStartDate"
+                        android:layout_width="match_parent"
+                        android:layout_height="wrap_content"
+                        android:layout_weight="0"
+                        android:text="@string/start_date_text"
+                        android:textAppearance="@style/TextAppearance.AppCompat.Body2"
+                        tools:ignore="InefficientWeight" />
+
+                    <EditText
+                        android:id="@+id/et_start_date"
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content"
+                        android:autofillHints=""
+                        android:ems="10"
+                        android:enabled="false"
+                        android:hint="@string/start_date_hint"
+                        android:inputType="text"
+                        tools:ignore="TextFields" />
```

Comment:
I think we should use a TextView for the date instead of a non-enabled EditText, if it is not editable by the user, it appears in grey and the user might want to click on it instead of setting the date with the buttons.

Reply:
Yep you are totally right :)